src/utils/ContainerUtil.js

```diff
@@ -33,10 +33,12 @@ var ContainerUtil = {
       var [dockerPort, portType] = key.split('/');
       var localUrl = null;
       var port = null;
-
       if (value && value.length) {
         port = value[0].HostPort;
       }
+      if (ip === 'docker.local') {
```

Comment:
May have to wait for this one - Current link preview is broken when VPN is enabled